src/ChattingList/ChattingListView.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from "react";
+import ChattingProfile from "./ChattingProfile";
+import ChattingListTopBar from "./ChattingListTopBar";
+import userProfileSet from "../constants/User";
+import styled from "styled-components";
+
+const Margin = styled.div`
+  height: 1.9vh;
+`;
+
+function ChattingList() {
+  const [matchUserSet, setMatchUserSet] = useState(userProfileSet);
+
+  function searchUser(searchUserInput) {
+    setMatchUserSet([]);
+    for (let userProfile of userProfileSet) {
+      if (userProfile.name.includes(searchUserInput)) {
+        setMatchUserSet((matchUser) => [...matchUser, userProfile]);
+      }
+    }
```

Comment:
```suggestion
    userProfileSet.forEach(elem => {
      if (elem.name.includes(searchUserInput)) {
        setMatchUserSet(matchUser => [...matchUser, elem]);
      }
    }
```

멋지게 `forEach`를 한번 써보자구요 ㅎㅎ 물론 이 방식도 좋습니다.